modules/raft/src/main/java/org/apache/ignite/raft/jraft/rpc/impl/AbstractClientService.java

```diff
@@ -233,7 +233,7 @@ public void complete(final Object result, final Throwable err) {
                         }
                     }
                     else {
-                        if (ThrowUtil.hasCause(err, null, PeerUnavailableException.class, ConnectException.class))
```

Comment:
We are trying to minimize changes to JRaft code. Do we really need to move the method?

Reply:
This method was introduced into `ThrowUtil` from JRaft. So, I decided to remove duplicates. If you think that it is a bad idea and we should not modify this code, then I will revert this change. No problems.

Reply:
No, I had no idea it was our code originally. I revoke my suggestion :)